### DISCUSSION TITLE
Development Preview 3

### DISCUSSION THREAD

**Discussion by FlipperPlz** [FlipperPlz/PakExplorer#3] [Announcements]
- Add Extraction Capabilities 
**Full Changelog**: https://github.com/FlipperPlz/PakExplorer/compare/preview-1...preview-2

<hr /><em>This discussion was created from the release <a href='https://github.com/FlipperPlz/PakExplorer/releases/tag/preview-2'>Development Preview 3</a>.</em>

**Comment by tacortz76** [comment]
@FlipperPlz Hello. Please excuse the unusual method of communication, unfortunately I did not find another one.
Could you help me out? I really need this application to learn some reforger scripts for dayz modding, but I have no way to compile it. Could you please release an .exe version for Windows and post/send it to me? Many thanks in advance!

**Comment by MolikoDeveloper** [comment]
https://hybrid-analysis.com/sample/804b1839b255e1999cec669b8bfdb2253698bad9e5de0fc91828d29ac6156a10

please confirm.

**Reply by FlipperPlz** [reply]
compile it yourself

**Reply by FlipperPlz** [reply]
I can attest to an unaltered compilation for the released binaries. If you still dont trust a random like me on the internet - compiling it yourself is the way.
